k8s/pkg/operation/assembly.go

```diff
@@ -154,12 +153,12 @@ func (ao *AssemblyOperation) buildLocalAssemblyJob(ctx context.Context, localObj
 			LocalAssemblyConfigTemplate.Namespace = pod.Namespace
 			LocalAssemblyConfigTemplate.StreamID = localObject.Spec.ObjectID
 			LocalAssemblyConfigTemplate.NodeName = localObject.Spec.Hostname
-			LocalAssemblyConfigTemplate.JobName = podLabels[schedulers.VineyardJobName]
+			LocalAssemblyConfigTemplate.JobName = podLabels["scheduling.k8s.v6d.io/job"]
 			LocalAssemblyConfigTemplate.TimeoutSeconds = timeout
 			if socket, err := ao.ResolveRequiredVineyarddSocket(
 				ctx,
-				podLabels[schedulers.VineyarddName],
-				podLabels[schedulers.VineyarddNamespace],
+				podLabels["scheduling.k8s.v6d.io/vineyardd"],
+				podLabels["scheduling.k8s.v6d.io/vineyardd-namespace"],
```

Comment:
Don't hard code strings. Use constant instead.

With constant variable, Go compiler will serve as a typo checker for you.

Reply:
Thanks for the catch. In fact, this change is to solve the problem of mutual import of golang packages, I will put all global variables into a golang package to solve it.